hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/KafkaMessageReceiverFactory.java

```diff
@@ -8,6 +8,7 @@
 import pl.allegro.tech.hermes.common.kafka.ConsumerGroupId;
 import pl.allegro.tech.hermes.common.kafka.KafkaNamesMapper;
 import pl.allegro.tech.hermes.common.metric.HermesMetrics;
+import pl.allegro.tech.hermes.consumers.config.ConsumerReceiverProperties;
```

Comment:
@Diewa @szczygiel-m 

I wonder how we can avoid introducing circular dependencies between packages. Currently, the `config` package is aware of other packages (let's call them domain packages). If we put classes with properties in the `config` package, then the domain packages will become aware of it. 

What do you think about putting classes with properties in the domain packages (just plain classes, without spring annotations) and registering them as beans, like this:

```java
@Bean
@ConfigurationProperties(prefix = "consumer.receiver")
ConsumerReceiverProperties consumerReceiverProperties() {
    return new ConsumerReceiverProperties()
}
```

We can discuss this idea tomorrow.